packages/components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -174,6 +174,7 @@ export const Tooltip = ({
                         variants={animationVariants}
                         animate={isShown ? 'shown' : 'hidden'}
                         transition={{ duration: 0.2, ease: 'easeInOut' }}
+                        onAnimationComplete={isShown ? () => {} : instance?.unmount}
```

Comment:
nitpick: I like this syntax more (not adding the prop at all if not needed)
```suggestion
                        {...(isShown ? {onAnimationComplete: instance?.unmount} : {})
```

Reply:
Honestly, it's harder to read imo 🙏